src/cubic_loader/qlik/ods_qlik.py

```diff
@@ -238,31 +238,49 @@ def cdc_verify_schema(self, dfm_object: str) -> None:
         :param dfm_object: S3 path of .dfm file that wil be used for verification
         """
         cdc_schema = dfm_schema_to_df(dfm_object)
-        cdc_name_set = set(cdc_schema.get_column("name"))
 
         # check dfm schema contains CDC_COLUMNS
-        assert set(CDC_COLUMNS).issubset(cdc_name_set)
+        assert set(CDC_COLUMNS).issubset(set(cdc_schema.get_column("name")))
         cdc_schema = cdc_schema.filter(pl.col("name").is_in(CDC_COLUMNS).not_())
 
         truth_schema = status_schema_to_df(self.etl_status)
-        truth_name_set = set(truth_schema.get_column("name"))
-        new_columns = cdc_schema.join(
```

Comment:
This previous "anit" join operation, that attempted to join on all columns was causing some major issues, resulting in attempting to create columns that already existed in DB tables.